readlif/reader.py

```diff
@@ -809,22 +830,23 @@ def __init__(self, filename):
             is_image_bool_list = self._recursive_memblock_is_image(self.xml_root)
             if False in is_image_bool_list:
                 from itertools import compress
```

Comment:
We should import at the top. I'm surprised our ruff configuration doesn't catch this.

Reply:
oof yes. same.